keyboards/ghs/xls/keymaps/via/keymap.c

```diff
@@ -0,0 +1,192 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*
+     * ┌───┬───┬───┬───┐
+     * │F13│F14│F15│F16│
+     * ├───┼───┼───┼───┤
+     * │Num│ / │ * │ - │
+     * ├───┼───┼───┼───┤
+     * │ 7 │ 8 │ 9 │   │
+     * ├───┼───┼───┤ + │
+     * │ 4 │ 5 │ 6 │   │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │   │
+     * ├───┴───┼───┤Ent│
+     * │ 0     │ . │   │
+     * └───────┴───┴───┘
+     */
+    [0] = LAYOUT_6x4(
+        KC_F13,  KC_F14,  KC_F15,  KC_F16,
+        KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS,
+        KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
+        KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+        KC_P0,   KC_P0,   KC_PDOT, KC_PENT,
+        KC_VOLD, KC_BRID, KC_DOWN, KC_PGDN,
+        KC_VOLU, KC_BRIU, KC_UP,   KC_PGUP
+    ),
+    [1] = LAYOUT_6x4(
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______
+    ),
+    [2] = LAYOUT_6x4(
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______
+    ),
+    [3] = LAYOUT_6x4(
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______
+    ),
+};
+
+keyevent_t encoder0_cw = {
```

Comment:
This is no longer accepted for new contributions, as <https://github.com/qmk/qmk_firmware/pull/17734> provides proper support.